OSPSuite.Presentation.Importer/Core/IUnformattedData.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using DevExpress.DataProcessing;
+
+namespace OSPSuite.Presentation.Importer.Core
+{
+   public interface IUnformattedData
+   {
+      IEnumerable<string> GetRow(int index);
+      IEnumerable<string> GetColumn(int columnIndex);
+      IEnumerable<string> GetColumn(string columnName);
+      Dictionary<string, ColumnDescription> Headers { get; }
+
+      IEnumerable<IEnumerable<string>> GetRows(Func<IEnumerable<string>, bool> filter);
+
+      bool AddRow(IEnumerable<string> row);
+      void AddColumn(string columnName, int columnIndex);
+      DataTable GetSheetAsDataTable();
+   }
+
+   public class UnformattedData : IUnformattedData
+   {
+      private readonly List<List<string>> _rawDataTable = new List<List<string>>();
+
+      public Dictionary<string, ColumnDescription> Headers { get; set; } =
```

Comment:
This should be readonly
Also we have a superclass called Cache that has a few more properties than the Dictionary. Maybe this could be used instead